commands/theme/preview.ts

```diff
@@ -1,35 +1,49 @@
 // @ts-nocheck
```

Comment:
`@hubspot/cms-dev-server` requires a version of Node incompatible with the one required in the CLI. Going to keep this comment in, until we can finally deprecate Node 18.

Reply:
I think it would make more sense to remove this and just add `@ts-ignore` comments to the lines that have problems. That way we still at least get some benefit from adding types to this